server/eclipse-project/pom.xml

```diff
@@ -4,9 +4,12 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>us.freeandfair.production</groupId>
 	<artifactId>colorado_rla</artifactId>
-	<version>0.0.1</version>
+	<version>0.7.0-alpha</version>
```

Comment:
That's fine. Just trying to keep the conversation going, and I wanted to test the release naming/launching process.
But this didn't seem like a particularly BAD PR to finally admit that we're nearing release, and can automatically test a bunch of stuff....
So I'll leave it in if I can get away with it :)